src/components/NavBar/NavBar.md

```diff
@@ -0,0 +1,41 @@
+# Nav Bar
+Provides a means of navigating to different areas with an application.
```

Comment:
Typo: "within"

I would also add the note about the collapsing of the nav to a hamburger menu on smaller screens (just to be explicit).